### PR TITLE
Rate checker design feedback fixes

### PR DESCRIPTION
**Pull request by sebworks**
Rate checker design feedback fixes GHE/19

## Testing

1. Visit `http://localhost:8000/owning-a-home/explore-rates/` and compare to mockup.

## Screenshot

- 
screen shot 2018-05-08 at 11 44 38 am
screen shot 2018-05-08 at 11 44 49 am

## Todos

-

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Browsers

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Internet Explorer 8, 9, 10, and 11
- [ ] Edge
- [ ] iOS Safari
- [ ] Chrome for Android

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

File: cfgov/jinja2/v1/owning-a-home/explore-rates/index.html [hunk 1]
```diff
@@ -308,7 +308,7 @@ <h3>Data table</h3>
                             <button class="chart-menu_btn
                                            cf-icon
                                            cf-icon__after
-                                           cf-icon-down">
+                                           cf-icon-download">
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/owning-a-home/explore-rates/index.html]: Let's update or delete this too https://github.com/cfpb/cfgov-refresh/blob/master/test/unit_tests/apps/owning-a-home/js/explore-rates/RateCheckerChartMenu-spec.js#L27

File: cfgov/jinja2/v1/owning-a-home/explore-rates/index.html [hunk 2]
```diff
@@ -536,11 +536,19 @@ <h4 class="next-steps-heading">Save for a larger down payment.</h4>
...
                             block__flush-top
                             block__padded-top
-                            note-ctr">
-                    <div class="note">
-                        <p class="alert"><strong>Check your credit report.</strong></p>
-                        If you haven’t checked your credit report recently, <a id="annualcreditreport" class="external-link" href="https://annualcreditreport.com" target="_blank" rel="noopener noreferrer">do so now</a>. If you find errors, <a id="ask-dispute-credit-report" href="/askcfpb/314/how-do-i-dispute-an-error-on-my-credit-report.html" target="_blank" rel="noopener noreferrer">get them corrected</a> <strong>before</strong> you apply for a mortgage.
-                    </div> <!-- /.note -->
+                            credit-note-ctr">
+                    <div class="credit-note
+                                m-notification m-notification__warning
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/owning-a-home/explore-rates/index.html]: Let's move `m-notification__warning` to its own line.

File: cfgov/jinja2/v1/owning-a-home/explore-rates/index.html [hunk 3]
```diff
@@ -536,11 +536,19 @@ <h4 class="next-steps-heading">Save for a larger down payment.</h4>
...
+                                m-notification__visible"
+                        role="alert">
+                        <span class="m-notification_icon cf-icon"></span>
+                        <div class="m-notification_content">
+                            <div class="h4 m-notification_message">Check your credit</div>
+                            <div class="h4 m-notification_explanation">
+                                If you haven’t checked your credit report recently, <a id="annualcreditreport" class="external-link" href="https://annualcreditreport.com" target="_blank" rel="noopener noreferrer">do so now</a>. If you find errors, <a id="ask-dispute-credit-report" href="/askcfpb/314/how-do-i-dispute-an-error-on-my-credit-report.html" target="_blank" rel="noopener noreferrer">get them corrected</a> before you apply for a mortgage.
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/owning-a-home/explore-rates/index.html]: Trivial. It would be helpful to wrap this long line.

Comment (anselmbradford) [reply cfgov/jinja2/v1/owning-a-home/explore-rates/index.html]: Is `id="annualcreditreport"` used?

Comment (anselmbradford) [reply cfgov/jinja2/v1/owning-a-home/explore-rates/index.html]: Also `id="ask-dispute-credit-report"`? Are they there as fragment identifiers in the URL for sharing or something?

File: cfgov/unprocessed/apps/owning-a-home/css/explore-rates.less [hunk 4]
```diff
@@ -564,21 +560,22 @@
...
-    .note-ctr {
+    .credit-note-ctr {
         border-top: 1px solid @gray-10;
     }
 
-    .note {
-        .grid_column( 8 );
-        margin: 0;
-        padding: 0 0 0 25px;
+    .credit-note {
```
Comment (anselmbradford) [inline cfgov/unprocessed/apps/owning-a-home/css/explore-rates.less]: This really should be a modifier on `m-notification`. We shouldn't have components that completely transform appearance without a modifier. Maybe update `.credit-note` to `.m-notification__borderless` or something and add a comment above it that it's an enhancement of `m-notification`. Eventually we'd migrate that into CF if this a sanctioned pattern that might find use outside of BAH.

File: cfgov/jinja2/v1/owning-a-home/explore-rates/index.html [hunk 5]
```diff
@@ -536,11 +536,19 @@ <h4 class="next-steps-heading">Save for a larger down payment.</h4>
...
-                        If you haven’t checked your credit report recently, <a id="annualcreditreport" class="external-link" href="https://annualcreditreport.com" target="_blank" rel="noopener noreferrer">do so now</a>. If you find errors, <a id="ask-dispute-credit-report" href="/askcfpb/314/how-do-i-dispute-an-error-on-my-credit-report.html" target="_blank" rel="noopener noreferrer">get them corrected</a> <strong>before</strong> you apply for a mortgage.
-                    </div> <!-- /.note -->
+                            credit-note-ctr">
+                    <div class="credit-note
+                                m-notification m-notification__warning
+                                m-notification__visible"
+                        role="alert">
+                        <span class="m-notification_icon cf-icon"></span>
+                        <div class="m-notification_content">
+                            <div class="h4 m-notification_message">Check your credit</div>
```
Comment (virginiacc) [inline cfgov/jinja2/v1/owning-a-home/explore-rates/index.html]: Should this be `Check your credit report`?

Comment (sebworks) [reply cfgov/jinja2/v1/owning-a-home/explore-rates/index.html]: I"m not sure, it use to have the word `report` but it was removed in the mockup.

File: cfgov/jinja2/v1/owning-a-home/explore-rates/index.html (hunk 1 above)
Comment (sebworks) [reply cfgov/jinja2/v1/owning-a-home/explore-rates/index.html]: fixed in https://github.com/cfpb/cfgov-refresh/pull/4100/commits/70e143a4407fbde245f74d2c9023ab588016c378.

File: cfgov/jinja2/v1/owning-a-home/explore-rates/index.html (hunk 2 above)
Comment (sebworks) [reply cfgov/jinja2/v1/owning-a-home/explore-rates/index.html]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/4100/commits/70e143a4407fbde245f74d2c9023ab588016c378.

File: cfgov/jinja2/v1/owning-a-home/explore-rates/index.html (hunk 3 above)
Comment (sebworks) [reply cfgov/jinja2/v1/owning-a-home/explore-rates/index.html]: Removed ids in https://github.com/cfpb/cfgov-refresh/pull/4100/commits/70e143a4407fbde245f74d2c9023ab588016c378.

File: cfgov/unprocessed/apps/owning-a-home/css/explore-rates.less (hunk 4 above)
Comment (sebworks) [reply cfgov/unprocessed/apps/owning-a-home/css/explore-rates.less]: Fixed Removed ids in 70e143a.

**Pull request merged**